coreplugins/cesiumion/public/components/FormikErrorFocus.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { connect } from "formik";
+
+class FormikErrorFocus extends React.Component {
+    isObject(value) {
```

Comment:
Think that there might me a more reliable implementation instead of rolling our own